operators/deploy/tenant-operator/values.yaml

```diff
@@ -19,6 +19,14 @@ configurations:
   maxConcurrentReconciles: 1
   sandboxClusterRole: crownlabs-sandbox
   tenantNamespaceKeepAlive: 168h
+  tenant:
+    resourcecaps:
+      # Maximum number of virtual cpus that can be requested by a tenant.
```

Comment:
I would write here whether those limits are *per instance* or *per tenant*.
If it is *per tenant*, those numbers look pretty small.

Reply:
I think @QcFe is going to override those values with ArgoCD or through CrownOps. But we can surely increase the default value. I forgot to add in the description that 0 means that the it no cap in resources, I'm going to add it.